src/controller_examples/rabbitmq_controller/exec/reconciler.rs

```diff
@@ -402,6 +403,18 @@ pub fn reconcile_core(rabbitmq: &RabbitmqCluster, resp_o: Option<Response<EmptyT
     }
 }
 
+fn make_labels(rabbitmq: &RabbitmqCluster) -> (labels: StringMap)
+    requires
+        rabbitmq@.metadata.name.is_Some(),
+    ensures
+        labels@ == rabbitmq_spec::make_labels(rabbitmq@),
+{
+    let mut labels = StringMap::empty();
+    labels.insert(new_strlit("app").to_string(), rabbitmq.name().unwrap());
```

Comment:
If you want every label to have this app=rabbitmq_name, you might want to switch the order of extend here: the argument of extend will overwrite whatever in self